doc/specs/vulkan/chapters/memory.txt

```diff
@@ -1721,6 +1721,7 @@ ifdef::VK_KHX_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances.
 endif::VK_KHX_device_group[]
+  * pname:ppData must: be a valid pointer to a pointer
```

Comment:
There's a hack in the script. It reuses the `"pointer to "` string and erases the last 4 char. I added "valid", so it would end up "valid pointer". Somebody should clean that up eventually, but yeah...